src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -165,11 +165,13 @@ def run(self, revision, review_request_id, diffset_revision):
             'modernize-use-auto',
             'modernize-use-default',
             'modernize-raw-string-literal',
-            'modernize-use-bool-literals',
```

Comment:
Actually, maybe it can be useful to comment out this rule instead of deleting it, with a brief explanation, e.g.:

                # 'modernize-use-bool-literals', # too noisy because of `while (0)` in many macros